docs/content/docs/build/experimental-features.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Experimental Features
+aliases:
+  - "/docs/build/experimental-features/"
+author: scottn12
+---
+
+## Overview
+
+At any given time, there are experimental features within FluidFramework that are still being actively developed/tested. In order to allow developers to adopt and test these features in a controlled environment, we introduced experimental feature gates to enable these features and easily disable them if they cause unexpected behavior.
+
+### Proceed with Caution
+
+Experimental features are by definition not fully supported and **should not be used in production scenarios**. Moreover, experimental features may be **added, modified, or removed without warning** in any minor or major release.
+
+## API
+
+To enable experimental features, you will be required to provide an implementation of `IConfigProviderBase`. Please note the following about `IConfigProviderBase`:
+
+-   `IConfigProviderBase` will accept input of a map-like object. The keys must be type `string` with the exact names of the features you want to enable. The values must be of type `ConfigType`, which is defined as a union of primitive types and arrays of primitive types.
```

Comment:
Where can a user find the list of currently supported experimental features? Is this enumerated in the API somewhere? In the docs?

Reply:
How will users know how to use them then? I get not broadcasting them super widely, but the idea is that customers will want to be able to opt into this stuff; I think we need to document what the options are _somewhere_, or no one will be able to use them.

Reply:
This kind of docs/education likely should reside in GH. For example if DDS wants to expose some experimental features they could likely document those as part of the DDS READMEs (for example). Im somewhat hesitant to expose specific features on ff.com as it implies prod readiness.

Reply:
That seems reasonable to me. Can we enumerate them in the package README then?